src/main/resources/db/migration/V1__create_breweries_table.sql

```diff
@@ -0,0 +1,18 @@
+CREATE EXTENSION IF NOT EXISTS postgis;
+CREATE EXTENSION IF NOT EXISTS pg_trgm;
+
+CREATE SCHEMA openbrewerydb
+
+CREATE TABLE IF NOT EXISTS openbrewerydb.breweries (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    brewery_type VARCHAR,
+    city VARCHAR NOT NULL,
+    state VARCHAR NOT NULL,
+    country VARCHAR NOT NULL,
+    location geography(POINT,4326) NOT NULL,
+    created_at TIMESTAMPTZ NOT NULL,
+    updated_at TIMESTAMPTZ NOT NULL
+);
```

Comment:
Should any of these fields be optional?

Reply:
I think all of these should be required, but I'm open.

Also, something I'd appreciate your opinion on, what should be the unique ID? I've been thinking about using a Yelp model (ex. `name-of-business-city` with any additional added like `name-of-business-city-2`). Thoughts?

Reply:
Ohhh, interesting question. When I looked at the current OBDB APIs, the id is specified by a number so that's why I picked the `SERIAL` type.

Honestly, your question kinda touches into what's the current source of truth of data. Is it the github repo dataset? Or is it a database?

My vision is that the DB is the source of truth. You already use AWS RDS so you it's incredibly safe when it comes to redundancy, backup etc. I imagine some UIs where people can propose breweries and we can do QA on if the proposal is good. If so, we would "approve" it which commits to the DB. 

I don't really know much or have context on the yelp model, so not sure of the advantages/disadvantages that it provides. Could look into it, but for the purposes of this project, I recommend we just go with the integer identifier for now and we can iterate on it.

Reply:
I agree that they are all required, but I didn't know if your current data set all has city, state, and country. Hence why I posed the question. I'll make them all required for now. :)

Reply:
Originally, my vision was that the Github repo is the source of truth for the same redundancy, backup, versioning, distribution, free hosting, etc. things and also a decent version control to approve changes. 

BUT, I'm running into usability issues (who wants to fork, edit in Excel, open PR, blah blah blah too many steps). So, I'm coming around to perhaps just sticking with the DB and building a CRUD system.

Reply:
If that's the case, we'll want to build authentication/authorization. Only a specific set of people (you) should be allowed to CUD.